geowebcache/core/src/main/java/org/geowebcache/config/GeoWebCacheConfiguration.java

```diff
@@ -276,6 +278,7 @@ public void setFormatModifiers(List<FormatModifier> formatModifiers) {
     
     /**
      * Returns the chosen lock provider
+     * @see ServerConfiguration#getLockProvider()
      * @return
```

Comment:
No return value documented.
For most of the stuff in GeoWebCacheConfiguration, probably just fine to have `@see ServerConfiguration#methodName()` and no other javadocs, unless it differs in some way from the interface behavior